docs/user_guide.md

```diff
@@ -402,3 +402,37 @@ you can see the item named "flink-pod-monitor" in the "Service Discovery" sectio
 ### Manage savepoints
 
 See this [doc](./savepoints_guide.md) on how to manage savepoints with the operator.
+
+### Flink Job update
```

Comment:
s/Flink Job update/Update Flink clusters and jobs?

Reply:
Done.